templates/get_started.md

```diff
@@ -1,16 +1,24 @@
 # Get started with Lean
 
-We have installation instructions for the following operating systems:
+For users wishing to get started quickly and easily, the [quickstart guide](https://lean-lang.org/lean4/doc/quickstart.html) should walk you through the installation procedure if using the Visual Studio Code editor is acceptable.
```

Comment:
Can these be explicitly marked as "Step 1: Install VSCode and Lean" (is it possible to make the "lean-lang.org" link pop up in another window, so this window isn't lost -- since they'll need to return to it to know how to make a Mathlib project), and "Step 2: Create a Project Using Mathlib", followed by the stated instructions?